src/drawmembrane/draw_membrane2a.c

```diff
@@ -770,10 +819,21 @@ int main(int argc, char *argv[])
   Membrane.x0_p = x0_p;          /* centre of the protein */
   Membrane.y0_p = y0_p;
   Membrane.z0_p = z0_p;
+  Membrane.dimer = False;         /*CPA hack*/
+
+  /* Manually add parameters for the second protomer. */
+  if (dimer){
+    printf("Dimer system detected. Turning on the hack mode.");
```

Comment:
Maybe call it "symmetric dimer" mode? "Hack" sounds as if it's not really supposed to be here.